### DISCUSSION TITLE
Multiboot *BSD and Linux?

### DISCUSSION THREAD

**Discussion by pauldrobertson** [zbm-dev/zfsbootmenu#466] [Questions]
Is it possible to add and boot FreeBSD and GhostBSD or OpenBSD along with a couple of Linux distros?  If so, how would I go about it?

**Comment by classabbyamp** [comment]
ZBM can only boot Linux as it uses `kexec` to boot into the selected operating system, but you could use something like rEFInd to choose whether to boot into BSD or ZBM

**Reply by pauldrobertson** [reply]
Thanks for the quick answer.  I know there’s ongoing LinuxBoot work in FBSD, perhaps that will align in the near future.
